stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -69,7 +147,176 @@ private static String getIdAttribute(View element) {
 
   @Override
   public View getViewForHighlighting(Object element) {
-    return (View)element;
+    return (View) element;
+  }
+
+  @Override
+  protected void onGetStyles(View element, StyleAccumulator styles) {
+
+    for (ViewCSSProperty property : getProperties()) {
+      try {
+        getCSSStringFromValue(
+            element,
+            property.getCSSName(),
+            property.getValue(element),
+            property.getAnnotation(),
+            styles);
+      } catch (IllegalAccessException | InvocationTargetException e) {
+        LogUtil.e("failed to get the style of element id=" + getIdAttribute(element));
```

Comment:
1. Don't forget to pass the exception into `LogUtil.e`
2. You should include something about the property (the name should work :-) ). Otherwise if 10 properties on 1 element fail we'll get 10 identical things printed to logcat. Also, if only 1 of them fails then it'll be easier to debug if we know which property isn't working.